fs.go

```diff
@@ -138,6 +139,25 @@ func (fs *stdFs) AccessTimes(name string) (rt, wt time.Time, err error) {
 	return atime.Get(fi), fi.ModTime(), nil
 }
 
+func (fs *stdFs) Stat(name string) (FileInfo, error) {
+	stat, err := os.Stat(name)
+	if err != nil {
+		return FileInfo{}, err
+	}
+
+	return FileInfo{
+		FileInfo: &fileInfo{
+			name:     name,
+			size:     stat.Size(),
+			fileMode: stat.Mode(),
+			isDir:    stat.IsDir(),
+			sys:      stat.Sys(),
+			wt:       stat.ModTime(),
+		},
```

Comment:
return FileInfo{FileInfo: stat, Atime: atime.Get(stat)}

Reply:
Thanks, overlooked this point